src/soter/openssl/soter_rsa_key_utils.h

```diff
@@ -100,4 +100,31 @@ static bool is_mod_size_supported(int mod_size)
     }
 }
 
+#if OPENSSL_VERSION_NUMBER < 0x10100000L
+/* Simple implementation for OpenSSL <1.1.0 where this function is missing */
+static int BN_bn2binpad(const BIGNUM* a, unsigned char* to, int tolen)
```

Comment:
Oh, there is no common header file to avoid duplication 😞 Well, not a big deal. Rule of three.

Reply:
hm, I remember that we always followed rule of two ) if it repeats second time then it is a signal to refactor it and move to common file
so, can we do not duplicate it and place in the soter_bignum_utils.h? or in some common file? because if we will need to update something, we should keep in mind that we have same copy in the another file...
now we should update `memset` -> `soter_wipe` in both places...